OctopusDSC/Tests/cOctopusServer.Tests.ps1

```diff
@@ -211,17 +211,26 @@ try
                         { Test-ParameterSet -Ensure 'Present' -State 'Installed' -DownloadUrl "blah2" -WebListenPrefix "blah3"} | Should not throw
                     }
                     It "Should not throw if 'OctopusAdminCredential' not supplied" {
-                        { Test-ParameterSet -Ensure 'Present' -State 'Installed' -DownloadUrl "blah2" -WebListenPrefix "blah3" -SqlDbConnectionString "blah4" -OctopusAdminCredential $null} | Should not throw
+                        { Test-ParameterSet -Ensure 'Present' -State 'Installed' -DownloadUrl "blah2" -WebListenPrefix "blah3" -OctopusAdminCredential $null } | Should not throw
                     }
                     It "Should not throw if 'OctopusAdminCredential' is '[PSCredential]::Empty'" {
-                        { Test-ParameterSet -Ensure 'Present' -State 'Installed' -DownloadUrl "blah2" -WebListenPrefix "blah3" -SqlDbConnectionString "blah4"} | Should not throw
+                        { Test-ParameterSet -Ensure 'Present' -State 'Installed' -DownloadUrl "blah2" -WebListenPrefix "blah3" } | Should not throw
```

Comment:
We don't have a default for `OctopusAdminCredential` on `Test-ParameterSet` now, so you'll have to explicitly pass it, wont you?

Reply:
Not on `Test-ParameterSet` directly but `Get\Set\Test-TargetResource` have it set to `[PSCredential]::Empty` and pass it to `Test-ParameterSet`.

Reply:
Makes sense... But can we always assume that? Someone might change those functions later without knowing the fact that this function relies on it... (not too fussed, to be honest - will leave it up to you)